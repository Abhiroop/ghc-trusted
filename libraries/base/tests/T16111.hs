module Main (main) where

import Data.Bits
import Data.Word

main :: IO ()
main = print $ toInteger (shiftL 1 hm :: Word64)
            == toInteger (shiftL 1 hm :: Word64)

hm :: Int
hm = -1
{-# NOINLINE hm #-}

