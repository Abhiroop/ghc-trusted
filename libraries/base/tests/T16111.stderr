T16111: arithmetic overflow

